turbine/src/broadcast_stage.rs

```diff
@@ -52,7 +53,8 @@ pub(crate) mod broadcast_utils;
 mod fail_entry_verification_broadcast_run;
 mod standard_broadcast_run;
 
-const CLUSTER_NODES_CACHE_NUM_EPOCH_CAP: usize = 8;
+const_assert_eq!(CLUSTER_NODES_CACHE_NUM_EPOCH_CAP, 5);
```

Comment:
Why the assert? Are you worried about max leader schedule stakes adjusting up? If so, maybe we should min this with some constant cap?

Reply:
Just as an explicit signal that if that constant is changed, it has ramifications here.
Also serves as a documentation that what the capacity actually is.